operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -160,14 +200,6 @@ The following is an example of an input file to use with the [Cray CLI](../../gl
 
 Refer to [Manage a Session Template](Manage_a_Session_Template.md) for more information about creating a session template.
 
-## Creating a BOS session using the new template
-
-(`ncn-mw#`) The new CPS-based session template can be used when creating a BOS session. The following is an example of creating a reboot session using the CLI:
-
-```bash
-cray bos v2 sessions create --template-name cps_rootfs_template --operation reboot
-```
-
 ## Appendix: `root=` kernel parameter
 
 This section supplies additional information about how BOS constructs the `root=` kernel parameter. This section does not require any
```

Comment:
It looks an incomplete sentence.Pls check.

Reply:
This sentence is a complete sentence. It was just continued on the next line. This renders fine in mark-up as a single line.

This section does not require any
action to be taken. It is merely supplemental information.